drivers/vdpa/virtio/virtio_vdpa.c

```diff
@@ -2106,6 +2106,12 @@ virtio_vdpa_dev_probe(struct rte_pci_driver *pci_drv __rte_unused,
 		pthread_mutex_init(&iommu_domain->domain_lock, NULL);
 	}
 
+	if (iommu_domain->container_ref_cnt == RTE_MAX_VFIO_GROUPS) {
```

Comment:
if vm_uuid is NULL, we will never reach this limit?

Reply:
yes. for VF that has null vm_uuid, it will have its own iommu domain and no other VFs can find the domain and use. So the container_ref_cnt will be 0 or 1.